### ISSUE TITLE
Implement location based auth in dor-rights-auth

### ISSUE THREAD

**Issue by jkeck** [sul-dlss/dor-rights-auth#8]
Referencing https://github.com/sul-dlss/media/issues/55. dor-right-auth needs to include \<location> restrictions as part of the authorization information it interprets and returns. 

As part of reflecting the rights to a caller, this module might also be part of determining if rights are OR'd together or ANDed in order to support these alternate scenarios:

Only Stanford community and only at kiosks (AND):
```
<access type="read">
   <machine>
      <group>stanford</group>
      <location>spec</location>
   </machine>
</access>
```
Stanford anywhere, world at kiosks (OR):
```
<access type="read">
   <machine>
      <group>stanford</group>
   </machine>
</access>
<access type="read">
   <machine>
      <location>spec</location>
   </machine>
</access>
```

**Comment by ndushay** [comment]
addressed by #10 and released as gem version 1.1.0

**Comment by jmartin-sul** [comment]
closing as per @ndushay's comment above, feel free to reopen if i'm doing that in error.
